universal-login-react/src/App.tsx

```diff
@@ -124,6 +124,7 @@ export const App = () => {
                   <hr/>
                   <EmojiForm
                     deployedWallet={new DeployedWallet(TEST_CONTRACT_ADDRESS, 'bob.mylogin.eth', TEST_PRIVATE_KEY, sdk)}
+                    onConnectionSuccess={() => { console.log('connect'); }}
```

Comment:
Hmmm

Reply:
Its just info for developers, who will test it in @react playground. Should I do some modal for this purpose?